main/webapp/modules/core/styles/common.css

```diff
@@ -348,6 +347,10 @@ a img {
   margin: 0.4em;
 }
 
+#file-input {
+  margin-bottom: 8px;
+}
```

Comment:
That's better, but still not a specific CSS selector: your `file-input` element id is very generic, so there could be other file inputs with the same id in other parts of the application (even though ids are supposed to be unique), and this CSS rule would apply to them. Why not just put inline CSS on the element?

Reply:
Done.